tests/vmi_configuration_test.go

```diff
@@ -269,6 +269,7 @@ var _ = Describe("Configurations", func() {
 				Expect(domXml).To(ContainSubstring("driver name='vhost' queues='3'"))
 			})
 
+			/* Commenting out the following test as CPU request is always set at the moment
```

Comment:
Let's remove it. It is in the git history and will for sure get outdated and ignored by devs.

Reply:
Will do